src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/InitialConsiderationHelper.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import uk.gov.hmcts.et.common.model.ccd.items.DateListedTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.HearingTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.JurCodesTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.HearingType;
+import uk.gov.hmcts.ethos.replacement.docmosis.domain.referencedata.JurisdictionCode;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.DocumentHelper.getHearingDuration;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Helper.nullCheck;
+
+public class InitialConsiderationHelper {
+
+    static final String RESPONDENT_NAME =
+            "| Respondent name given | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|In Et1 by claimant | %s|\r\n"
+                    + "|In Et3 by Respondent | %s|";
+
+    static final String HEARING_DETAILS =
+            "|Hearing Details | |\r\n"
+                    + "|-------------|:------------|\r\n"
+                    + "|Date | %s|\r\n"
+                    + "|Type | %s|\r\n"
+                    + "|Duration | %s|";
+
+    static final String HEARING_MISSING = String.format(HEARING_DETAILS, "", "", "");
+    static final String RESPONDENT_MISSING = String.format(RESPONDENT_NAME, "", "");
+
+    private InitialConsiderationHelper() {

```

Comment:
Let's make this a service bean, not a utility class.

Reply:
Done